### ISSUE TITLE
mAP is low and no detection result.

### ISSUE THREAD

**Issue by bei91** [meituan/YOLOv6#158]
Hi, using your own data for training, after 180 epochs, the accuracy is relatively low, and the inference has no result.
![Picture1](https://user-images.githubusercontent.com/16322361/176581044-e6611377-d5d0-4b25-976b-b8e6c4335735.png)

**Comment by mtjhl** [comment]
Hi, which dataset are you using?

**Comment by binven1** [comment]
I also met the problem that all the mAP is low，but yolov5lite is normal

**Comment by Zephyr69** [comment]
Does it train normally with yolov5?

With the latest repo the mAP results on my small target dataset are still not on par with a v5 model of the same scale and the training is extremely unstable, but the difference is also not that big. 

It could also be inappropriate hyperparameters. But we need to see more information from the training process to get an idea about where to tune.

**Comment by bei91** [comment]
@mtjhl Thank you for your reply, i use your own datasets, not a public dataset.

**Comment by SongHfei** [comment]
> Hi, which dataset are you using?

i use the widerface and myself own datasets,reduce the flops to 60M,The accuracy of the validation set is very poor：


Inferencing model in val datasets.: 100%|##############################################################################| 64/64 [01:19<00:00,  1.24s/it]

Evaluating speed.

Evaluating mAP by pycocotools.
Saving ./runs/train/yolov6_face_66M_0705/predictions.json...
loading annotations into memory...
Done (t=0.11s)
creating index...
index created!
Loading and preparing results...
DONE (t=2.71s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=36.83s).
Accumulating evaluation results...
DONE (t=5.91s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.001
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.001
Epoch: 60 | mAP@0.5: 0.00012148454109214603 | mAP@0.50:0.95: 1.989929560401417e-05

**Comment by mtjhl** [comment]
拉取一下最新的代码，在非coco 数据上，把这个 is_coco 设置为 False
https://github.com/meituan/YOLOv6/blob/fb9485fdb3c5e290c51b8c7dc8b05e2bc78a8da3/data/coco.yaml#L9

**Comment by SongHfei** [comment]
> 拉取一下这个最新的代码，在非coco数据上，把is_coco设置为False
> 
> https://github.com/meituan/YOLOv6/blob/fb9485fdb3c5e290c51b8c7dc8b05e2bc78a8da3/data/coco.yaml#L9

试过了．

**Comment by Chilicyy** [comment]
hi @sssssshf If dataset has been trained with yolov5/7 or older version of yolov6, the dataset cache might be incompatible for yolov6. Add --check-images and --check-images to create new cache.
